ros/src/computing/perception/detection/packages/lidar_tracker/nodes/obj_fusion/obj_fusion.cpp

```diff
@@ -123,151 +151,203 @@ static void fusion_objects(void) {
   pub_msg.header = header;
   autoware_msgs::CloudClusterArray cloud_clusters_msg;
   cloud_clusters_msg.header = header;
+  visualization_msgs::MarkerArray marker_array_msg;
+  int marker_id = 0;
 
-  for (unsigned int i = 0; i < obj_label_current.obj_id.size(); ++i) {
-    jsk_recognition_msgs::BoundingBox bounding_box;
-    if (obj_indices.at(i) == -1)
-      continue;
-
-    v_cloud_cluster_current.at(obj_indices.at(i)).label = object_type;
+  for (unsigned int i = 0; i < obj_label.obj_id.size(); ++i) {
+    if (obj_indices.at(i) == -1) continue;
 
+    v_cloud_cluster.at(obj_indices.at(i)).label = object_type;
     if (object_type == "car") {
-      v_cloud_cluster_current.at(obj_indices.at(i)).bounding_box.label = 0;
+      v_cloud_cluster.at(obj_indices.at(i)).bounding_box.label = 0;
```

Comment:
I would suggest creating an enum type for the labels which will also add semantics to it.